subprojects/testfx-core/src/main/java/org/testfx/util/DebugUtils.java

```diff
@@ -16,31 +16,48 @@
  */
 package org.testfx.util;
 
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Set;
 import java.util.function.Function;
+import java.util.function.Supplier;
 
 import javafx.event.Event;
+import javafx.geometry.Bounds;
+import javafx.geometry.Rectangle2D;
 import javafx.scene.Node;
+import javafx.scene.image.Image;
 import javafx.scene.input.KeyCode;
 import javafx.scene.input.MouseButton;
+import javafx.stage.Screen;
+import javafx.stage.Window;
 
 import org.hamcrest.Matcher;
 import org.testfx.api.FxRobot;
+import org.testfx.api.FxService;
 import org.testfx.api.FxToolkit;
+import org.testfx.service.support.CaptureSupport;
 import org.testfx.service.support.FiredEvents;
 
 /**
- * Utility class for displaying additional info or running code when a test fails using
- * {@link org.testfx.api.FxAssert#verifyThat(Node, Matcher)} or its related methods.
+ * Utility class for displaying additional info, running code, or capturing an image of the test when
+ * a test fails using {@link org.testfx.api.FxAssert#verifyThat(Node, Matcher)} or its related methods.
  *
  * <p>
  *     Quickly chain things together using {@link #compose(Function[])} or use standard handlers like
  *     {@link #informedErrorMessage(String, boolean, FxRobot, boolean, boolean)}. Any {@code indent} parameters
  *     in methods are there to specify what to insert to offset values. Default indent value is three spaces
  *     (e.g. {@code "   "}).
  * </p>
+ * <p>
+ *     When a test fails, an image of the it can be captured and saved to a PNG file. Supports the size
```

Comment:
"When a test fails, an image of the it" => "When a test fails a screenshot of the scene can be captured and saved to a PNG file." Not sure about the rewording, but there is a typo with "of the it".

Reply:
Thanks for the catch on the typos. It was caused by editing it after writing something else originally. I was having trouble deciding how to word things, and I like you wording much better.

Reply:
On second thought, I think I'll just remove the typo "the." The concept I want to introduce/begin with in the paragraph's first sentence is the idea of capturing an image of the test and later clarify what kinds of images can be captured. I don't want to say screenshot initially because it starts to leads people astray as to what is supported.